contracts/facets/PartyBPositionActions/PartyBPositionActionsFacetImpl.sol

```diff
@@ -30,50 +26,56 @@ library PartyBPositionActionsFacetImpl {
 		require(!accountLayout.suspendedAddresses[msg.sender], "PartyBFacet: Sender is Suspended");
 		require(!appLayout.partyBEmergencyStatus[quote.partyB], "PartyBFacet: PartyB is in emergency mode");
 		require(!appLayout.emergencyMode, "PartyBFacet: System is in emergency mode");
-		LibMuonPartyB.verifyPairUpnlAndPrice(upnlSig, quote.partyB, quote.partyA, quote.symbolId);
+
 		accountLayout.partyANonces[quote.partyA] += 1;
 		accountLayout.partyBNonces[quote.partyB][quote.partyA] += 1;
 
 		currentId = LibPartyBPositionsActions.openPosition(quoteId, filledAmount, openedPrice);
-		uint256[] memory quoteIds = new uint256[](1);
-		uint256[] memory filledAmounts = new uint256[](1);
-		uint256[] memory marketPrices = new uint256[](1);
-		quoteIds[0] = quoteId;
-		filledAmounts[0] = filledAmount;
-		marketPrices[0] = upnlSig.price;
-		LibSolvency.isSolventAfterOpenPosition(
-			quoteIds,
-			filledAmounts,
-			marketPrices,
-			upnlSig.upnlPartyB,
-			upnlSig.upnlPartyA,
-			quote.partyB,
-			quote.partyA
-		);
+		if (accountLayout.bindState[quote.partyA].partyB != quote.partyB) {
```

Comment:
How is the binding supposed to work here? I thought partyA simply cannot have other positions with partyBs. How could this condition ever be true?